runtime/pack/dist/opt/cfilter/plugin/cfilter.vim

```diff
@@ -1,23 +1,25 @@
 " cfilter.vim: Plugin to filter entries from a quickfix/location list
-" Last Change: 	May 12, 2018
-" Maintainer: 	Yegappan Lakshmanan (yegappan AT yahoo DOT com)
-" Version:	1.0
+" Last Change: Aug 23, 2018
+" Maintainer: Yegappan Lakshmanan (yegappan AT yahoo DOT com)
+" Version: 1.1
```

Comment:
Should have been included in https://github.com/neovim/neovim/pull/9172/commits/2c7ed420d93fdefe5734f2e9cf6b863119d412f3.

Update the commit message that this adds the missing cfilter change. Mention the vim commit hash that updated cfilter.

Reply:
Updated